Makefile

```diff
@@ -34,10 +34,12 @@ TAG?=$(shell echo "$$(date +v%Y%m%d)-$$(git describe --always --dirty)")
 # the full image tag
 IMAGE?=$(REGISTRY)/$(IMAGE_NAME):$(TAG)
 PLATFORMS?=linux/amd64,linux/arm64
+PROGRESS?=auto
```

Comment:
note: this is the default https://docs.docker.com/reference/cli/docker/buildx/build/